src/db_impl.h

```diff
@@ -167,6 +169,7 @@ class TitanDBImpl : public TitanDB {
   friend class BaseDbListener;
   friend class TitanDBTest;
   friend class TitanThreadSafetyTest;
+  friend class TitanCompactionFilterFactory;
```

Comment:
This is useless now?

Reply:
`TitanCompactionFilterFactory` need to access `blob_file_set_` field in `CreateCompactionFilter` method.